cylc/flow/scripts/remove.py

```diff
@@ -18,7 +18,27 @@
 
 """cylc remove [OPTIONS] ARGS
 
-Remove one or more task instances from a running workflow.
+Remove tasks from a running workflow and the workflow's history.
```

Comment:
IMO as a headline "remove a task from a running workflow" is still pretty ambiguous and kind of harks back to the old task-pool focused `cylc remove` . Can we can focus on the essential new functionality?
```suggestion
Erase the run-history of tasks, allowing them to run again in the same flow.
```

Reply:
Ok that reads good to me. What does @oliver-sanders think?

Reply:
Maybe ambiguous was the wrong word. 

It's more that I think exactly what it means to "remove tasks from a running workflow" won't be at all obvious to users (e.g. remove tasks from the graph?, remove upcoming tasks - like pre-spawned ones in Cylc 7 - so they won't run? or what?) and it doesn't really suggest the important new history erasure aspect. 

Actually we should add this (or similar) to the main text as well:
```
This command can also be used to remove final status incomplete tasks from the n=0 active window, to prevent or recover from a stall, if they don't need to be completed to continue to the workflow.
```

I think "erase the run-history of tasks" can be viewed as covering all of the above including the task pool operations, if you think of "run-history" as all of task evolution at run time, not just jobs.

Reply:
> I think "erase the run-history of tasks" can be viewed as covering the task pool operations too?

I don't think that is obvious personally as it's mixing task spawning (internal concept) with run history (which means job submissions to users).

Reply:
"Task instances" makes perfect sense to me, because it corresponds directly to task proxy objects in the task pool. But I think users will still wonder what in hell an instance is.

We should probably just refer to the user-facing active window (n=0) concept here.

Anyhow, let's move this conversation to Ronnie's side PR.